staging/src/k8s.io/legacy-cloud-providers/azure/azure_client.go

```diff
@@ -129,10 +123,18 @@ type StorageAccountClient interface {
 	Create(ctx context.Context, resourceGroupName string, accountName string, parameters storage.AccountCreateParameters) *retry.Error
 	Delete(ctx context.Context, resourceGroupName string, accountName string) *retry.Error
 	ListKeys(ctx context.Context, resourceGroupName string, accountName string) (result storage.AccountListKeysResult, rerr *retry.Error)
-	ListByResourceGroup(ctx context.Context, resourceGroupName string) (result storage.AccountListResult, rerr *retry.Error)
+	ListByResourceGroup(ctx context.Context, resourceGroupName string) (result []storage.Account, rerr *retry.Error)
```

Comment:
@andyzhangx Interface `ListByResourceGroup` has been changed to return a list of accounts, which is much easier to consume.

Reply:
good to know, let's see the azure test result